internal/services/db/db.go

```diff
@@ -282,13 +289,23 @@ func (db *DB) PushTokenTableExists(suffix string) (bool, error) {
 }
 
 // TransferName returns the name of the transfer db for the given contract
-func (d *DB) TransferName(contract string) string {
-	return fmt.Sprintf("%v_%s", d.chainID, strings.ToLower(contract))
+func (d *DB) TransferName(contract string) (string, bool) {
```

Comment:
This should probably just return `errors.New("Bad contract address")`. And then we can do the standard:

```
name, err := d.TransferName(ev.Contract)
		if err  != nil {
			return err
		}
```

Reply:
I would also suggest renaming this method to something that matches what it does (my fault for giving it a confusing name).

How about `TableNameSuffix`?